<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureChat</class>
 <widget class="QDialog" name="ConfigureChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="ChatLabel">
     <property name="text">
      <string>Select the chat</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="ChatComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="CancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="DeleteChatButton">
     <property name="text">
      <string>Delete chat</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="SendInviteLabel">
     <property name="text">
      <string>Select the contact to send invite</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QComboBox" name="ContactComboBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="SendInviteButton">
     <property name="text">
      <string>Send invite</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLineEdit" name="RenameLineEdit">
     <property name="placeholderText">
      <string>New chat name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="RenameChatButton">
     <property name="text">
      <string>Rename the chat</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLabel" name="SuccessLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
